.travis.yml

```diff
@@ -0,0 +1,63 @@
+sudo: false
+
+language: php
+
+php:
+    - 7.2
+    - 7.3
+    - nightly
+
+before_install:
+    - phpenv config-rm xdebug.ini || true
+    - if [ "$SYMFONY_VERSION" != "" ]; then composer require --dev --no-update symfony/symfony=${SYMFONY_VERSION}; fi
+
+install:
+    - travis_retry composer install --classmap-authoritative --no-suggest --prefer-dist
+
+script:
+    - vendor/bin/phpunit
+
+jobs:
+    include:
+
+        - stage: Test
+          env: SYMFONY_VERSION='4.0.*'
+          php: 7.2
+
+        - stage: Test
+          env: SYMFONY_VERSION='3.4.*'
+          php: 7.2
+
+        - stage: Test
+          env: DEPENDENCIES=low
+          php: 7.2
+          install:
+              - travis_retry composer update --classmap-authoritative --no-suggest  --prefer-dist --prefer-lowest --prefer-stable
+
+        - stage: Code Quality
+          env: CODING_STANDARDS
+          script:
+              - vendor/bin/phpcs -p
+
+        - stage: Code Quality
+          env: STATIC_ANALYSIS
+          script:
+              - vendor/bin/phpstan analyse
```

Comment:
I have to work on templating of boilerplate as this is going to come up again and again

Reply:
I'm not sure a layer of indirection is that useful. Something that could analyse the state of a repo could be (eg checking testing frameworks, Travis config, license year, repo's configuration).

Reply:
Could tell you what is not aligned, but then how easy it is to propagate a bugfix like a `docker-compose.yml` gotcha or a new `php-cs-fixer/`phpstan` configuration?